src/components/homePage/Search/__tests__/Search.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import Search from "../Search";
+
+describe("Search component", () => {
+  test("renders Search", () => {
```

Comment:
rename to "Search should render correctly"

Reply:
fixed